src/peft/tuners/lora/model.py

```diff
@@ -249,7 +261,8 @@ def _create_and_replace(
             if adapter_name not in self.active_adapters:
                 # adding an additional adapter: it is not automatically trainable
                 new_module.requires_grad_(False)
-            self._replace_module(parent, target_name, new_module, target)
+
+            self._replace_module(parent=parent, child_name=target_name, new_module=new_module, child=target)
```

Comment:
Although I'm not against using keywords here, this method is called numerous times in PEFT with just positional args, so let's leave it at that for consistency.